src/main/java/uk/gov/justice/hmpps/prison/api/resource/ServiceAgencySwitchesResource.kt

```diff
@@ -38,4 +41,26 @@ class ServiceAgencySwitchesResource(private val service: ServiceAgencySwitchesSe
     serviceCode: String,
   ): List<PrisonDetails> =
     service.getServicePrisons(serviceCode)
+
+  @ApiResponses(
+    ApiResponse(responseCode = "200", description = "OK"),
+    ApiResponse(responseCode = "401", description = "A valid auth token was not presented", content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]),
+    ApiResponse(responseCode = "403", description = "The auth token does not have the necessary role", content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]),
+    ApiResponse(responseCode = "404", description = "The service code does not exist", content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]),
+    ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request", content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))]),
+  )
+  @Operation(summary = "Retrieve a list of prisons switched on for the service code")
+  @PreAuthorize("hasRole('SERVICE_AGENCY_SWITCHES')")
+  @PostMapping("/{serviceCode}/prison/{prisonId}")
+  @ResponseStatus(HttpStatus.CREATED)
+  @Hidden()
```

Comment:
I take it `Hidden` is only temporary - otherwise there is no point with all the swagger doc stuff